aws/resource_aws_lb.go

```diff
@@ -316,7 +316,8 @@ func resourceAwsLbUpdate(d *schema.ResourceData, meta interface{}) error {
 
 	attributes := make([]*elbv2.LoadBalancerAttribute, 0)
 
-	if d.HasChange("access_logs") {
+	// Access Logs are not supported for Network Loadbalancers
+	if d.Get("load_balancer_type").(string) != "network" && d.HasChange("access_logs") {
```

Comment:
I'm not feeling comfortable in allowing a `terraform apply` when the code will just ignore it. I do think we should add a guard when planning/applying, saying that this is not allowed rather than ignoring it, which may people think that logs are configured whereas it is not.

Thoughts? 😄

Reply:
I do agree, but preventing `apply` entirely might cause some issues as well. For my use case, see: https://github.com/terraform-aws-modules/terraform-aws-alb/blob/master/main.tf#L15. In certain modules it wouldn't be possible to create `network` load balancers this way.
But I'm a bit of a TF noob. Is there a way to do a check within the module?

Otherwise we could check against the `access_logs.enabled` parameter.

What do you think?

Reply:
We could also add a `log.Printf("[WARN]")`, but I'm not sure what the preferred method is for handling these cases.

Reply:
@Ninir Any thoughts?

Reply:
Let's go for an INFO notice here :)

Reply:
@Ninir Done!